golang/cosmos/x/vtransfer/ibc_middleware_test.go

```diff
@@ -765,15 +764,15 @@ func (s *IntegrationTestSuite) TestHops() {
 					expectedRecords := []swingsettypes.InboundQueueRecord{}
 					if tc.senderIsTarget {
 						expectedRecords = append(expectedRecords, swingsettypes.InboundQueueRecord{
-							Action: &vibckeeper.WriteAcknowledgementEvent{
+							Action: &vibctypes.AcknowledgementPacketEvent{
```

Comment:
This is a little bug fix.  The code hasn't yet been exercised for sender targets since they haven't been implemented.